dandi/utils.py

```diff
@@ -686,3 +688,30 @@ def is_url(s):
     TODO: redo
     """
     return s.lower().startswith(("http://", "https://", "dandi://"))
+
+
+def get_module_version(module: Union[str, "Module"]) -> object:
```

Comment:
`"Module"` should be `types.ModuleType`, and the return value should be `Any`.

Reply:
done -- made it into
```
def get_module_version(module: Union[str, types.ModuleType]) -> Optional[str]:
```
assuming that those modules we deal with would have it as a `str`